### ISSUE TITLE
Install instructions don't display -g for preferGlobal packages

### ISSUE THREAD

**Issue by itsananderson** [npm/newww#960]
Packages that are marked as `preferGlobal: true` are displayed with the `npm install <package>` instructions in the top-right of the package page. It'd be great if they were displayed with `npm install -g <package>` instead.

![image](https://cloud.githubusercontent.com/assets/516559/7504514/bb613182-f3ff-11e4-89d7-00381c73e8c2.png)

Based on this presenter code, it looks like it should be showing `-g` for packages marked as `preferGlobal`.

https://github.com/npm/newww/blob/739be64a160fd5e71f4d32412ed8d9d41a75a108/presenters/package.js#L89

Based on that code, I wonder if maybe the `preferGlobal` option is getting filtered out somewhere earlier in the pipeline, because otherwise I'd expect it to show up.

**Comment by mklement0** [comment]
Since #1017 is now closed as a duplicate, let me quote from a response there by @zeke on June 3  to provide some background:

> It's an issue in the new registry. The new private API that the website consumes is not including the preferGlobal property in its response object.

The fact that this seemingly easy-to-fix bug has been ignored for such a long time may indicate that focus is shifting away from _global_ packages, which would be regrettable: I, for one, primarily publish CLI packages intended for global use - npm is a great multi-platform distribution mechanism for that.

**Comment by aredridel** [comment]
There are  also other priorities and the database has to expose the info :)

**Comment by mklement0** [comment]
@aredridel: Understood. I was trying to provide (impersonal) exegesis (so to speak), followed by a (personal) expression of unhappiness with the perceived trend, because I love npm's ability to install CLIs globally, complete with `man` pages.

**Comment by aredridel** [comment]
Yeah. We've no antagonism toward `-g`. That's gonna work into the future!

**Comment by mklement0** [comment]
@aredridel: Awesome; glad to hear it.
